common/servlet/src/main/java/io/apiman/common/servlet/HttpCacheUtil.java

```diff
@@ -0,0 +1,19 @@
+package io.apiman.common.servlet;
```

Comment:
Can you add the standard copyright notice to the top of this file?  It should be this:

{code}
/*
 * Copyright 2016 JBoss Inc
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
{code}